### ISSUE TITLE
[Suggestion] KDE Plasma Mobile GUI

### ISSUE THREAD

**Issue by TheAlienDrew** [EXALAB/AnLinux-App#250]
Since you're making a way to install linux distros and desktop environments on phones, wouldn't it be a good idea to support KDE plasma mobile? It would especially be useful for the GUI.

**Comment by exalab999** [comment]
Yes, this was already in plan and passed the test, it will be available in
the update this month, but please note that KDE is heavy and not very
friendly for old device.

**Comment by TheAlienDrew** [comment]
That's fine with me, I got a high-end device.

**Comment by RasheedAZ** [comment]
Is there an update on this? I think on Fedora KDE, just installing [plasma-mobile](https://src.fedoraproject.org/rpms/plasma-mobile) package sets up the mobile GUI.
